packages/docs/docs/docs/deploying.md

```diff
@@ -26,49 +24,55 @@ Use `npm` to create a `package.json` file:
 npm init
 ```
 
-This command will ask you for details about the project. For this very basic
-guide you can just press enter to accept the default values of each field.
+This command will ask you for details about the project. For this getting started project, just press **Enter** to accept the default values for each field.
 
-Then, install ZeppelinOS in your project folder by running:
+Then install ZeppelinOS in your project folder by running:
 
 ```console
-npm install zos
+npm install zos@2.3.0-rc.2
 ```
 
-Run `npx zos --help` to get a list of all the ZeppelinOS commands available. At any time
-during these guides, when we introduce a new command, run
-`npx zos <command-name> --help` to get more information about that command and
-its arguments.
+Run `npx zos --version` to output the current ZeppelinOS version.
```

Comment:
Should we add a CTA on this, such as verifying that it matches the installed one?

Reply:
What do you mean by 'CTA'?

Reply:
Call-to-action

Reply:
Sure, we could add "Make sure the current version matches the version you installed."